app/assets/stylesheets/application.scss

```diff
@@ -50,3 +50,13 @@ a:visited {
  .govuk-header__navigation-item a {
      color: white !important;
  }
+
```

Comment:
Slightly surprised that this isn't available already - maybe a comment would be appropriate?